package NYT.NTableClient.NProto;

import "yt/ytlib/chunk_client/proto/data_source.proto";

import "yt/ytlib/controller_agent/proto/job.proto";

////////////////////////////////////////////////////////////////////////////////

message TTablePartitionCookie
{
    optional NChunkClient.NProto.TDataSourceDirectoryExt data_source_directory = 1;
    repeated NControllerAgent.NProto.TTableInputSpec table_input_specs = 2;
    optional string user = 3;
}

////////////////////////////////////////////////////////////////////////////////
